uniffi_bindgen/src/bindings/python/templates/EnumTemplate.py

```diff
@@ -19,7 +19,12 @@ def __init__(self):
     # Each enum variant is a nested class of the enum itself.
     {% for variant in e.variants() -%}
     class {{ variant.name()|enum_variant_py }}:
-        def __init__(self,{% for field in variant.fields() %}{{ field.name()|var_name }}{% if loop.last %}{% else %}, {% endif %}{% endfor %}):
+        {% for field in variant.fields() %}
+            {{- field.name()|var_name }}: "{{- field|type_name }}";
```

Comment:
Ah, regarding this, I added the semicolon here so that the generator doesn't have to worry about keeping track of indentation and all of that, all of the type hints for instance variables would just occupy a single line and be separated by semicolon, which I believe is allowed by Python. I'm using Python's Black formatter and it's able to separate these type hints into multiple lines.

I'm now looking into removing the semicolon and replacing it with a new line with proper indentation.

Reply:
Oh, if it is to save some work on handling whitespace and indentation then I don't mind, I thought this was left in accidentally.

Reply:
Fantastic, thanks!